<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_produit</class>
 <widget class="QMainWindow" name="gestion_produit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="sel_col">
    <property name="geometry">
     <rect>
      <x>587</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>id</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>type</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>quantité</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>état</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>prix</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>status</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="tableau_produit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>401</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="rech">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Recherche</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Form</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>status</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>source</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="source">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>type</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="quantite">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="status">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>            --</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En preparation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En cours de livraison</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En attente de livraison</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Livre</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ISBN</string>
     </property>
    </widget>
    <widget class="QComboBox" name="type">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>            --</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>produits alimentaire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>produits scolaire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>produits informatique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>produits domestiques</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="etat">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>            --</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>quantite maximal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>epuise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>presque epuise</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>quantité</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="prix">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prix</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>état</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>stat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exporter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>export Excel</string>
    </property>
   </widget>
   <widget class="QLabel" name="produit_error_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pave">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher pavé</string>
    </property>
   </widget>
   <widget class="QLabel" name="dechet_error_lb">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pave_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher temperature</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
